reference/7.3/Microsoft.PowerShell.Core/About/about_ANSI_Terminals.md

```diff
@@ -174,33 +163,65 @@ The following members control how or when ANSI formatting is used:
+  > [about_Experimental_Features][3] and [Using experimental features][4].
 
 ## Cmdlets that generate ANSI output
 
-- The markdown cmdlets - the
-  [Show-Markdown](xref:Microsoft.PowerShell.Utility.Show-Markdown) cmdlet
-  displays the contents of a file containing markdown text. The output is
-  rendered using ANSI sequences to represent different styles. You can manage
-  the definitions of the styles using the
-  [Get-MarkdownOption](xref:Microsoft.PowerShell.Utility.Get-MarkdownOption)
-  and
-  [Set-MarkdownOption](xref:Microsoft.PowerShell.Utility.Get-MarkdownOption)
-  cmdlets.
+- The markdown cmdlets - the [Show-Markdown][5] cmdlet displays the contents of
+  a file containing markdown text. The output is rendered using ANSI sequences
+  to represent different styles. You can manage the definitions of the styles
+  using the [Get-MarkdownOption][6] and [Set-MarkdownOption][7] cmdlets.
 - PSReadLine cmdlets - the PSReadLine module uses ANSI sequences to colorize
   PowerShell syntax elements on the command line. The colors can be managed
-  using [Get-PSReadLineOption](xref:PSReadLine.Get-PSReadLineOption) and
-  [Set-PSReadLineOption](xref:PSReadLine.Set-PSReadLineOption).
-- `Get-Error` - the [Get-Error](xref:Microsoft.PowerShell.Utility.Get-Error)
-  cmdlet provide a detailed view of an **Error** object, formatted to make it
-  easier to read.
-- `Select-String` - Beginning with PowerShell 7.0,
-  [Select-String](xref:Microsoft.PowerShell.Utility.Select-String) uses ANSI
-  sequences to highlight the matching patterns in the output.
+  using [Get-PSReadLineOption][8] and [Set-PSReadLineOption][9].
+- `Get-Error` - the [Get-Error][10] cmdlet returns a detailed view of an
+  **Error** object, formatted to make it easier to read.
+- `Select-String` - Beginning with PowerShell 7.0, [Select-String][11] uses
+  ANSI sequences to highlight the matching patterns in the output.
 - `Write-Progress` - ANSI output is managed using `$PSStyle.Progress`, as
-  described above. For more information, see
-  [Write-Progress](xref:Microsoft.PowerShell.Utility.Write-Progress)
+  described above. For more information, see [Write-Progress][12]
+
+## Redirecting output in `Host` mode
+
+By default, `$PSStyle.OutputRendering` is a set to **Host**. The ANSI escape
+sequences are removed from redirected or piped output.
+
+**OutputRendering** only applies to rendering in the Host, `Out-File`, and
+`Out-String`. Output from native executables isn't affected.
+
+PowerShell 7.3 changed the behavior of `Out-File` and `Out-String` for the
+following scenarios:
+
+- When the input object is pure string, these cmdlets keep the string unchanged
+  regardless of the **OutputRendering** setting.
+- When the input object needs to have a formatting view applied to it, these
+  cmdlets keep or remove escape sequences from the formatting output strings
+  based on the **OutputRendering** setting.
+
+This is a breaking change in these cmdlets compared to PowerShell 7.2.
```

Comment:
One more thing -- this change was backported to 7.2.6, so now 7.2 has the same behavior.